pallets/schemas/src/lib.rs

```diff
@@ -236,7 +247,12 @@ pub mod pallet {
 		) -> DispatchResult {
 			let sender = ensure_signed(origin)?;
```

Comment:
`create_shema` is still here, it just creates a schema with empty settings, so I'm not sure what you mean backward compatibility. It seems like you have it already.

Reply:
I'll schedule a huddle time to talk about your concerns here.